pkg/filesystem/block_device_backed_file_pool_test.go

```diff
@@ -309,4 +310,12 @@ func TestBlockDeviceBackedFilePool(t *testing.T) {
 		sectorAllocator.EXPECT().FreeList([]uint32{0, 0, 0, 0, 0, 0, 0, 0, 5})
 		require.NoError(t, f.Close())
 	})
+
+	t.Run("WriteAt", func(t *testing.T) {
+		f, err := pool.NewFile()
+		require.NoError(t, err)
+
+		_, err = f.WriteAt([]byte{0}, -1)
+		testutil.RequireEqualStatus(t, status.Error(codes.InvalidArgument, ""), err)
```

Comment:
Did this test pass like this? We aren’t supposed to return an empty message, right?

Reply:
Oops- I assumed it was just comparing the code, updated.